src/main/java/org/mule/extension/db/internal/domain/connection/derby/DerbyConnectionParameters.java

```diff
@@ -117,4 +118,9 @@ public String getPassword() {
   public String getUser() {
     return null;
   }
+
+  @Override
+  public java.util.Optional<TlsContextFactory> getTlsContextFactory() {
+    return java.util.Optional.empty();
```

Comment:
Does derby support TLS? Is this change aligned that in a near future we may be adding TLS Support for every connection provider?

Reply:
No idea, but if it does by implementing this method we would offer the feature.